src/python/pants/backend/python/providers/python_build_standalone/rules.py

```diff
@@ -118,6 +121,24 @@ class PBSPythonProviderSubsystem(Subsystem):
         ),
     )
 
+    _release_constraints = StrOption(
+        default=None,
+        help=textwrap.dedent(
+            """
+            Version constraints on the PBS "release" version to ensure only matching PBS releases are considered.
+            Constraints should be specfied in a similar manner to interpreter constraints: e.g., `>=1.2.3,<2`.
```

Comment:
nit: Maybe use version strings in the example that look YYYYMKMDD PBS "releases"?

Reply:
Good point. Will do.

Reply:
Done.